backend/typescript/services/implementations/camperService.ts

```diff
@@ -803,7 +804,7 @@ class CamperService implements ICamperService {
   }
 
   /* eslint-disable consistent-return */
-  async inviteWaitlistedCamper(waitlistedCamperId: string): Promise<any> {
+  async inviteWaitlistedCamper(waitlistedCamperId: string): Promise<unknown> {
```

Comment:
do we have to change this?

Reply:
it kept giving warnings and it annoyed me lol so I removed.